integration_tests/pages/page.ts

```diff
@@ -27,7 +27,7 @@ export default abstract class Page {
           { id: 'color-contrast', enabled: false },
         ],
       })
-      cy.checkA11y(null, null, this.terminalLog)
+      cy.checkA11y(null, null, this.terminalLog, true)
```

Comment:
Adding this means the test still passes despite an A11y violation. It still lists the error in the console etc.
There's a violation in the MOJ pagination macro, which we can't alter in our repo.